samples/bluetooth/broadcast_audio_source/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -9,4 +9,21 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
 # LC3 lib requires floating point support in the c-lib NEWLIB is one way of getting that.
 CONFIG_NEWLIB_LIBC=y
 CONFIG_MAIN_STACK_SIZE=4096
+
+# USB Audio related configs
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr Broadcast Source"
+CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=n
+CONFIG_USB_DEVICE_AUDIO=y
+
+# Two streams in one subgroup
+CONFIG_BT_ISO_MAX_CHAN=2
+CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=2
+CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
+
+# Audio buffer
+CONFIG_RING_BUFFER=y
```

Comment:
Are any of this nRF5340 specific? 

I was wondering if this should be moved a USB overlay config file if it works work other platforms too, although I guess it has a dependency on LIBLC3?

Reply:
IMO, the USB part only really makes sense if it also has LIBLC3 to encode.  For now, I don't know other targets that has USB device support and are powerful enough for the LC3 codec (and can do LE Audio broadcast).  Maybe this can be extended to the nrf5340dk too

Reply:
Indeed. 

How LC3 is enabled in the future may also change as per this PR: https://github.com/zephyrproject-rtos/zephyr/pull/64704/files#diff-7e388e16f1a780c43a46235fe6fda25c2cd5abbc9e2d9614621356e1fc8dd065R23-R29

We may be able to use the same approach for the USB stuff, so that we automatically enable it for all nRF5340s